### ISSUE TITLE
TypeError for a form with grouped fields

### ISSUE THREAD

**Issue by joene789** [verbb/formie#633]
**Description**
Submitting a form with grouped fields or (re)saving a submission for this form throws a TypeError:
`Argument 2 passed to verbb\formie\helpers\Variables::getParsedValue() must be an instance of verbb\formie\elements\Submission or null, instance of verbb\formie\elements\NestedFieldRow given, called in /***/vendor/verbb/formie/src/fields/formfields/Hidden.php on line 126`

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.23
- Craft version: 3.7.18.2
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.23",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.24](https://github.com/verbb/formie/releases/tag/1.4.24)
